### ISSUE TITLE
Updater again

### ISSUE THREAD

**Issue by vomikan** [moneymanagerex/moneymanagerex#2444]
Seems Pre-release only looks like Unstable.

![image](https://user-images.githubusercontent.com/6836805/80768792-083dc100-8b54-11ea-8851-ecfb3ef254c0.png)

-  Check MMEX version
  - [x] 1.3.5

**Comment by gabriele-v** [comment]
I think RC are anyway Pre-Releases, so it should enough to flag also RC as Pre-Releases in GitHub, or I'm missing something?

**Comment by vomikan** [comment]
If any (beta, alpha, RC) release in GH published as Release (not Pre-release) it would be marked as Stable.

**Comment by gabriele-v** [comment]
Yep, that's correct. So I don't understand the issue.. Or issue means that it should look at alpha/beta/rc and not GitHub Pre-release status? Up to me now it has the correct behavior

**Comment by vomikan** [comment]
Exactly. But I'll prefer to leave it as is.
